Src/Model/Adapter/AdapterInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace Aot\Model\Adapter;
+
+interface AdapterInterface
```

Comment:
:clap: 
I'm aplausing, ur first interface! However, an interface has to constraint a class and must be correctly documented

Reply:
done